app/views/spree/shared/_google_analytics.html.erb

```diff
@@ -1,39 +1,36 @@
 <% if tracker = current_tracker %>
+  <script>
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
 
-  <%= javascript_tag do %>
-    var _gaq = _gaq || [];
-    _gaq.push(['_setAccount', '<%= tracker.analytics_id %>']);
-    _gaq.push(['_trackPageview']);
+    ga('create', '<%= tracker.analytics_id %>', 'auto');
+    ga('require', 'displayfeatures');
+    ga('send', 'pageview');
 
-    <% if flash[:commerce_tracking] && @order.present? %>
-      <%# more info: https://developers.google.com/analytics/devguides/collection/gajs/methods/gaJSApiEcommerce %>
-      _gaq.push(['_addTrans',
-        "<%= j @order.number %>",
-        "<%= tracker.store.name %>",
-        "<%= @order.total %>",
-        "<%= @order.adjustments.tax.sum(:amount) %>",
-        "<%= @order.adjustments.shipping.sum(:amount) %>",
-        "<%= j @order.bill_address.city %>",
-        "<%= j @order.bill_address.state_text %>",
-        "<%= j @order.bill_address.country.name %>"
-      ]);
+    <% if @order && order_just_completed?(@order) %>
```

Comment:
Turns out it is necessary because the `order_just_completed?` helper is not available on all pages. Which is a silly way to handle it, but it's how core is currently doing it so I'm fine with us borrowing that logic as well.